frontend/containers/sdgs/constants.ts

```diff
@@ -3,3 +3,23 @@ export const SDGS_SIZES = {
   small: 88,
   large: 110,
 };
+
+export const SDGS_IMAGES = {
```

Comment:
The images the only thing that isn't returned by the enums. This way we can just do `SDGS_IMAGES[id]`